comp_loinc/build.py

```diff
@@ -1,12 +1,20 @@
-import typer
-from typing import List
-from ingest.part_ingest import PartOntology
-from ingest.code_ingest import CodeIngest
+"""CLI"""
 import os
 import subprocess
 
+import typer
+
+try:
```

Comment:
The `try` block handles cases where the CLI is being called as a module/package: `python -m comp_loinc`
The `except` block handles cases where CLI is being called directly from python file: `python comp_loinc/build.py`

I normally see CLIs designed like to be run as a module w/ `-m`, so normally I don't have/see these try/except blocks. But @putmantime I figured you might want to continue running `build.py` as you have, so I added this try/except block to help w/ that.